### PR TITLE
etcdserver: leader latency optimization

### PR DESCRIPTION
**Pull request by xiang90**
/cc @heyitsanthony @gyuho

### CODE REVIEW

File: etcdserver/raft.go [hunk 1]
```diff
@@ -198,6 +201,13 @@ func (r *raftNode) start(s *EtcdServer) {
 					return
 				}
 
+				// the leader can write to its disk in parallel with replicating to the followers and them
```
Comment (gyuho) [inline etcdserver/raft.go]: `...can write to its disk while concurrently replicating to the followers, and then followers can write to their disks`?

**Comment by gyuho** [comment]
lgtm.

File: etcdserver/raft.go (hunk 1 above)
Comment (xiang90) [inline etcdserver/raft.go]: this is directly copied from the thesis. :) so i do not really want to change.

opinion? @heyitsanthony

Comment (gyuho) [inline etcdserver/raft.go]: Got it. Then makes sense to me. Thanks!

**Comment by heyitsanthony** [comment]
lgtm

**Pull request merged**